introduction.md

```diff
@@ -1,2 +1,5 @@
 Hello, I am Benjamin!
-I love Ruby path at Code School!
+* Java
+* C/C++
+* Android
+* Git
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?